client/app/cloud/components/project/billing/instance-list/billing-instance-list.component.html

```diff
@@ -1,94 +1,72 @@
+              data-popover-class="details-popover"
+              data-popover-is-open="$ctrl.DetailsPopoverService.isCurrentDetails($row)"
+              data-popover-placement="top"
+              data-popover-append-to-body="true"
+              data-popover-trigger="none"
+              data-ng-click="$ctrl.DetailsPopoverService.setCurrentDetails($row)"></span>
+    </oui-column>
+    <oui-column data-title=":: $ctrl.instanceColTotal | translate" data-property="total" data-sortable>
+    </oui-column>
+    <oui-column data-ng-if="$ctrl.showSwitchToMonthlyBillingOption || $ctrl.showAdditionalInstanceDetails">
+        <span data-ng-if="$row.monthlyBilling && $row.monthlyBilling.status === 'ok'"
+              data-translate="cpbc_hourly_instance_monthly_billing_since"
+              data-translate-values="{ since : ($row.monthlyBilling.since | date:'shortDate') }">
+        </span>
+        <span data-ng-if="$row.monthlyBilling && $row.monthlyBilling.status === 'activationPending'">
+            {{ :: 'cpbc_hourly_instance_monthly_billing_activationPending' | translate }}
+            <i class="fa fa-clock-o left-space-m8"></i>
+        </span>
+        <span class="opacity6"
+              data-ng-if="!$row.monthlyBilling && $row.isDeleted"
+              data-translate="cpbc_hourly_instance_deleted">
+        </span>
 
-                    <!-- CONFIRM PASSING TO MONTHLY PAYMENT -->
-                    <div data-ng-show="instanceConsumption.instanceId === $ctrl.instanceToMonthly"
-                            class="text-right animate-fadeInRight-enter animate-fadeOutLeft-leave full-width cell-confirm">
+        <!-- CONFIRM PASSING TO MONTHLY PAYMENT -->
+        <div data-ng-if="$row.instanceId === $ctrl.instanceToMonthly"
+             class="text-right animate-fadeInRight-enter animate-fadeOutLeft-leave full-width cell-confirm">
+            <button class="btn btn-default"
+                    type="reset"
+                    data-ng-click="$ctrl.endInstanceToMonthlyConversion()"
+                    data-ng-disabled="$ctrl.loaders.monthlyBilling">
+                <i class="glyphicon glyphicon-remove right-space-m8"></i>
+                <span data-translate="common_cancel"></span>
+            </button>
+            <button class="btn btn-primary"
+                    data-uib-tooltip="{{ :: 'cpbc_hourly_instance_pass_to_monthly_info' | translate }}"
+                    type="submit"
+                    data-ng-click="$ctrl.confirmMonthlyPaymentActivation()"
+                    data-ng-disabled="$ctrl.loaders.monthlyBilling">
+                <i class="fa fa-check right-space-m8"></i>
+                <span data-translate="cpbc_hourly_instance_pass_to_monthly_button"
+                    data-translate-values="{ price: $ctrl.data.instanceToMonthlyPrice.price.text }"></span>
+                <spinner data-ng-if="$ctrl.loaders.monthlyBilling">
```

Comment:
⚠️ Shouldn't be a [`<oui-spinner>`](https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/oui-angular/spinner) instead?